sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -223,6 +222,10 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       primaryContactLastName: institutionLocation.primaryContact.lastName,
       primaryContactEmail: institutionLocation.primaryContact.email,
       primaryContactPhone: institutionLocation.primaryContact.phoneNumber,
+      canadaPostalCode: institutionLocation.data.address.canadaPostalCode,
```

Comment:
I got the point that UI needs a second field to deal with the _Canadian/not Canadian_ postal code, but I do not believe that it should be reflected in the database.
IMO, DB should keep only `postalCode` and it is valid also for the `country`.

Reply:
when the submitted form is viewed again we need these values also need to be populated. in the case of Pincode, canadaPostalCode and otherPostalCode are different. Canada has a mask and the other is free-text. are you suggesting to calculate those in API or in vue side?

Reply:
It could be on API at the controller level, IMO.
Do you think that it is valid to save these additional fields on DB?
For instance, when generating the e-Cert files, I would like to avoid checking where the country is saved for instance. I would say that it would be better to have a single property on DB that tell us the country.